app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -2,37 +2,40 @@
   <fieldset ng-disabled="claimDisabled">
     <div ng-show="storageClasses | hashSize" class="form-group">
       <!--storage class-->
-      <label class="required">Storage Classes</label>
+      <label>Storage Class</label>
+      <div>
+      <ui-select
+        ng-if="storageClasses"
+        ng-model="claim.storageClass"
+        theme="bootstrap"
+        search-enabled="true"
+        title="Select a storage class"
+        class="select-role"
+        flex>
+        <ui-select-match placeholder="Select a storage class">
+          <span>
+            {{$select.selected.metadata.name}}
+          </span>
+        </ui-select-match>
+        <ui-select-choices
+          repeat="sclass as sclass in storageClasses | orderBy: 'metadata.name'">
+          <div>{{ sclass.metadata.name }}</div>
+          <div ng-if="sclass | annotation : 'description'">
+            <small>&nbsp;&nbsp;</small>
```

Comment:
Rather than non-breaking spaces, we should use CSS margins.

This can probably all be one `small` tag.